Assets/HoloToolkit/UX/Scripts/BoundingBoxes/BoundingBox.cs

```diff
@@ -172,8 +172,7 @@ protected set
 
         [Tooltip("Any renderers on this layer will be ignored when calculating object bounds")]
         [SerializeField]
-        [Range(0, 31)]
-        protected int ignoreLayer = 2;//Ignore Raycast
+        protected LayerMask ignoreLayers = LayerMask.NameToLayer("Ignore Raycast");
```

Comment:
NameToLayer might not be good.  Devs may have different layer names and may even have this missing in their projects.

Reply:
It should be okay in this case since "Ignore Raycast" is one of Unity's built-in layers, and since it's just a default value.